### DISCUSSION TITLE
V3 bitrate and FPS Boost

### DISCUSSION THREAD

**Discussion by AdiAbuAli** [mrlt8/docker-wyze-bridge#146] [Q&A]
Hi,
When I monitor live streaming data from Wyze app (or even synology surveillance station), I notice it can jump for some instances beyound the HD quality it fetch from wyze server (bitrate=120Kb & FPS=20), e.g. for some instances, FPS reach 22 or 23, and bitrate 155Kb, and then back to around 20 and 120Kb...
Will the bridge clip the bitrate and FPS during the streaming acccording to the ones fetched when container started, or it's just Camera itself boost for short instances and back to normal performance?
In other words, the bridge will be transparent in streaming performance?

**Comment by mrlt8** [comment]
From what I can tell, the cameras only produce one stream for everything. A change in the app/bridge will affect the _one_ stream.

AFAIK, bitrate/framerate changes should be passed onto the stream without any negatives, HOWEVER, changing the resolution/frame size seems to break HLS and RTMP streams, so [v0.6.4](https://github.com/mrlt8/docker-wyze-bridge/releases/tag/v0.6.4)+ will attempt to restart the stream if it receives X number of frames with the wrong frame size. This can be ignored (and potentially improve stability since the stream doesn't get killed) with `IGNORE_RES`.

**Comment by AdiAbuAli** [comment]
Yes am referring to same stream...sometime it boost in term of bitrate and FPS (not frame size)...anyways, am using RTSP/TCP.
Anyways, its good that bridge doesnt clip the FPS or bitrate if boosted beyound what it fetched once container started in RTSP.

Thanks,
________________________________
